Makefile

```diff
@@ -65,12 +65,11 @@ coverage: ## check code coverage quickly with the default Python
 		$(BROWSER) htmlcov/index.html
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/bigchaindb.rst
-	rm -f docs/modules.rst
-	sphinx-apidoc -o docs/ bigchaindb
```

Comment:
I don't think so... I think we've plugged into sphinx so that readthedocs will automatically generate any new documentation from the templates? @libscott might know better?